src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let number = days * 40;
+
+  if (days >= 7) {
+    number = number - 50;
+
+    return number;
```

Comment:
Одразу повертай значення і непотрібно буде переприсвоювати значення
```suggestion
    return number - 50;
```